Warning in library

Hello!
I found a warning in your library in the "DHT.cpp" file.
"comparison between signed and unsigned integer expressions"
I think that it can be fixed by replacing on line 29
`#define TIMEOUT -1`
 with
`#define TIMEOUT 0xffffffffUL`
or maybe 
`#define TIMEOUT UINT32_MAX`

Hello,
You can add these warnings :

> DHT_sensor_library\DHT.cpp:40:45:` warning: unused parameter 'count' [-Wunused-parameter]  DHT::DHT(uint8_t` pin, uint8_t type, uint8_t count) {

> DHT_sensor_library\DHT.cpp:282:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare] if (expectPulse(LOW) == TIMEOUT) {

> DHT_sensor_library\DHT.cpp:287:27: warning: comparison between signed and unsigned integer expressions [-Wsign-compare] if (expectPulse(HIGH) == TIMEOUT) {

> DHT_sensor_library\DHT.cpp:312:20: warning: comparison between signed and unsigned integer expressions [-Wsign-compare] if ((lowCycles == TIMEOUT) || (highCycles == TIMEOUT)) {

> DHT_sensor_library\DHT.cpp:312:47: warning: comparison between signed and unsigned integer expressions [-Wsign-compare] if ((lowCycles == TIMEOUT) || (highCycles == TIMEOUT)) {

--- a/DHT.cpp
+++ b/DHT.cpp
@@ -26,7 +26,7 @@
 #include "DHT.h"
 
 #define MIN_INTERVAL 2000 /**< min interval value */
-#define TIMEOUT -1        /**< timeout on */
+#define TIMEOUT UINT32_MAX /**< Used programmatically for timeout. Not a timeout duration. Type: uint32_t. */
 
 /*!
  *  @brief  Instantiates a new DHT class

--- a/DHT.cpp
+++ b/DHT.cpp
@@ -26,7 +26,8 @@
 #include "DHT.h"
 
 #define MIN_INTERVAL 2000 /**< min interval value */
-#define TIMEOUT UINT32_MAX /**< Used programmatically for timeout. Not a timeout duration. Type: uint32_t. */
+#define TIMEOUT UINT32_MAX /**< Used programmatically for timeout. \
+                                 Not a timeout duration. Type: uint32_t. */
 
 /*!
  *  @brief  Instantiates a new DHT class

--- a/DHT.cpp
+++ b/DHT.cpp
@@ -26,8 +26,9 @@
 #include "DHT.h"
 
 #define MIN_INTERVAL 2000 /**< min interval value */
-#define TIMEOUT UINT32_MAX /**< Used programmatically for timeout. \
-                                 Not a timeout duration. Type: uint32_t. */
+#define TIMEOUT                                                                \
+  UINT32_MAX /**< Used programmatically for timeout.                           \
+                   Not a timeout duration. Type: uint32_t. */
 
 /*!
  *  @brief  Instantiates a new DHT class

